src/models/output.ts

```diff
@@ -210,6 +233,44 @@ class Output {
 
     return [output, outputBuffer];
   }
+
+  /**
+   * Checks if the script length is within the valid limits
+   *
+   * @returns {boolean} True if the script is within valid limits
+   *
+   * @memberof Output
+   * @inner
+   */
+  hasValidLength(): boolean {
+    // No script can have more than the maximum length
+    if (this.script.length > MAXIMUM_SCRIPT_LENGTH) {
+      return false;
+    }
+
+    // P2PKH and P2SH scripts can only have 30 or 25 bytes (with or without timelock)
+    if (P2PKH.identify(this.script) || P2SH.identify(this.script)) {
```

Comment:
I think this check is already done in the `identify` methods in each class, so if it's not 30/25 length, then will return `false` for this `if`